Sources/Swift/Tools/UIImageHelper.swift

```diff
@@ -25,9 +25,8 @@ final class UIImageHelper {
         let blue = CGFloat(data[0]) / 255.0
         let green = CGFloat(data[1]) / 255.0
         let red = CGFloat(data[2]) / 255.0
-        let alpha = CGFloat(data[3]) / 255.0
```

Comment:
Does this mean that we could change line 23 to the following? I was able to run the full unit test suite without it crashing, anyways.
```
let data = pixelBuffer.bindMemory(to: UInt8.self, capacity: 3)
```

Reply:
Oh yes. We dont need to read all 4 bytes.